rust/connlib/clients/android/src/lib.rs

```diff
@@ -405,7 +405,13 @@ fn connect(
         handle,
     };
 
-    let session = Session::connect(api_url.as_str(), secret, device_id, callback_handler)?;
+    let session = Session::connect(
+        api_url.as_str(),
+        secret,
+        device_id,
+        callback_handler,
+        Duration::from_secs(5 * 60),
```

Comment:
Builder pattern would be nice here to avoid updating all the clients, but API compat isn't a big deal right now so it's fine

Reply:
created #2986